chapter1/AppCreationScripts/sample.json

```diff
@@ -58,6 +57,7 @@
         }
```

Comment:
Value should be **TaskUser** as per SPA config file.

Reply:
done!